sycl/doc/extensions/ExtendedAtomics/SYCL_INTEL_extended_atomics.asciidoc

```diff
@@ -113,389 +115,399 @@ In SYCL 1.2.1, support for floating-point types is limited to the +load+, +store
 
-=== Modify Listing 4.2
+|===
 
-==== From:
+The additional member functions below are available for atomic references to floating-point types:
 
-[source,c++]
-----
-/* Available only when: accessMode == access::mode::atomic && dimensions == 0 */
-operator atomic<dataT, access::address_space::local_space> () const;
+|===
+| Member Function | Description
 
-/* Available only when: accessMode == access::mode::atomic && dimensions > 0 */
-atomic<dataT, access::address_space::local_space> operator[](id<dimensions> index) const;
+| `T fetch_add(T operand, memory_order order = default_read_modify_write_order) const`
+| Atomically adds _operand_ to the value of the object referenced by this +atomic_ref+ and assigns the result to the value of the referenced object.  Returns the original value of the referenced object.
 
-/* Available only when: accessMode == access::mode::atomic && dimensions == 1 */
-atomic<dataT, access::address_space::local_space> operator[](size_t index) const;
-----
+| `T operator+=(T operand) const`
+| Equivalent to +fetch_add(operand)+.
 
-===== To:
+| `T fetch_sub(T operand, memory_order order = default_read_modify_write_order) const`
+| Atomically subtracts _operand_ from the value of the object referenced by this +atomic_ref+ and assigns the result to the value of the referenced object.  Returns the original  value of the referenced object.
 
-[source,c++]
-----
-/* Available only when: accessMode == access::mode::atomic && dimensions == 0 */
-operator intel::atomic<dataT, access::address_space::local_space> () const;
+| `T operator-=(T operand) const`
+| Equivalent to +fetch_sub(operand)+.
 
-/* Available only when: accessMode == access::mode::atomic && dimensions > 0 */
-intel::atomic<dataT, access::address_space::local_space> operator[](id<dimensions> index) const;
+| `T fetch_min(T operand, memory_order order = default_read_modify_write_order) const`
+| Atomically computes the minimum of _operand_ and the value of the object referenced by this +atomic_ref+, and assigns the result to the value of the referenced object. Returns the original value of the referenced object.
 
-/* Available only when: accessMode == access::mode::atomic && dimensions == 1 */
-intel::atomic<dataT, access::address_space::local_space> operator[](size_t index) const;
-----
+| `T fetch_max(T operand, memory_order order = default_read_modify_write_order) const`
+| Atomically computes the maximum of _operand_ and the value of the object referenced by this +atomic_ref+, and assigns the result to the value of the referenced object. Returns the original value of the referenced object.
 
-=== Modify Table 4.49
+|===
+
+The additional member functions below are available for atomic references to pointer types:
 
-==== Replace each instance of:
+|===
+| Member Function | Description
 
-+atomic+
+| `T* fetch_add(ptrdiff_t operand, memory_order order = default_read_modify_write_order) const`
```

Comment:
I guess `ptrdiff_t` should be `difference_type`, according to the sample header below

Reply:
Agreed.